spinalcordtoolbox/deepseg/models.py

```diff
@@ -249,8 +259,25 @@
                              'tandem with another model for specialized cord localisation of spinal cords with tumors '
                              '(https://github.com/ivadomed/findcord_tumor).',
          'url': 'https://github.com/ivadomed/model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel',
-         'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel']},
-    'seg_exvivo_gm-wm_t2':
+         'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel'],
+         'citation': textwrap.dedent("""
+             ```
```

Comment:
I don't know if this is supported by this particular markdown processor, but triple-backticks can be annotated with a language name to get syntax highlighting. See [this markdown reference](https://www.markdownguide.org/extended-syntax/#syntax-highlighting), and [the list of languages supported by github](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting), which includes python, json, bibtex.

Reply:
It looks like this doesn't really matter? Seems like the processor auto-detects bibtex highlighting, as we get the following output regardless of the presence of `bibtex`:

![image](https://github.com/user-attachments/assets/b93f2742-5d6d-4d03-8847-318c972bb1f9)

Still, it might be worth adding anyway, since it documents what format the citation is?